components/ActionNetworkForm.vue

```diff
@@ -140,14 +144,16 @@
         <div v-if="isGDPRCountry" class="form-group opt-in-wrapper">
           <label>{{ $t('gdpr.opt_in_label') }}</label>
           <div class="form-check">
-            <input class="form-check-input" type="radio" name="opt_out" :value="false" v-model="optedOut" required id="gdpr_yes">
-            <label class="form-check-label" for="gdpr_yes">
+            <label class="form-check-label">
```

Comment:
The ID attributes here were breaking things when more than one form was on the same page (as with the test page).